src/Microsoft.AspNetCore.Mvc.Core/AuthorizeFilter.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
Is there a Filters namespace this can go to? Not sure we want to pollute the top-level namespace with this. Is there a place where most other filters live?